internal/pkg/template/templates/workloads/partials/cf/envvars-common.yml

```
@@ -29,6 +29,8 @@
       {{- end}}
 {{- end}}{{- end}}
 {{- if eq .WorkloadType "Load Balanced Web Service"}}
+{{- if not .HTTPDisabled}}
```

Consider flipping the field name https://refactoring.com/catalog/removeDoubleNegative.html
```suggestion
{{- if .HTTPEnabled}}
```